software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/couchdb/CouchDBNode.java

```diff
@@ -37,8 +38,15 @@
 @ImplementedBy(CouchDBNodeImpl.class)
 public interface CouchDBNode extends SoftwareProcess, WebAppService {
 
+    @SetFromFlag("downloadUrl")
+    AttributeSensorAndConfigKey<String, String> DOWNLOAD_URL = ConfigKeys.newSensorAndConfigKeyWithDefault(SoftwareProcess.DOWNLOAD_URL,
+            "http://www-eu.apache.org/dist/couchdb/source/${version}/apache-couchdb-${version}.tar.gz");
```

Comment:
From Apache site: `Please use the backup mirrors only to download PGP and MD5 signatures to verify your downloads or if no other mirrors are working.`